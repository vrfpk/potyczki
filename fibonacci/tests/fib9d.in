45309496888687753
