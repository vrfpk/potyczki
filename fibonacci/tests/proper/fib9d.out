150000000000001667
